opentelekomcloud/acceptance/ces/resource_opentelekomcloud_ces_alarmrule_test.go

```diff
@@ -50,6 +51,19 @@ func TestCESAlarmRule_basic(t *testing.T) {
 	})
 }
 
+func TestAccCheckCESV1AlarmValidation(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { common.TestAccPreCheck(t) },
+		ProviderFactories: common.TestAccProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config:      testCESAlarmRuleValidation,
+				ExpectError: regexp.MustCompile("Error: either `alarm_actions` or `ok_actions` should be specified+"),
```

Comment:
I suppose, it's a bit not regex you want to have :) 
https://regex101.com/r/52NA6v/1

`d+` in the end basically meaning one ore more `d` characters.

Reply:
Thank you for noticing :)